src/components/input/InputComponent.tsx

```diff
@@ -14,7 +14,7 @@ export const InputText:React.FC<InputProps> = (props) => (
         style={inputStyles.inputContainer}
```

Comment:
This file name is not good, because we using "Component" suffix that are no need here ... (because we in component folder).